server_management/management/commands/update.py

```diff
@@ -250,15 +250,15 @@ def handle(self, *args, **options):
                         remote['server'].get('settings_file', 'production')
                     )):
 
-                        sudo('pip install -q gunicorn', user=project_folder)
                         sudo('[[ -e requirements.txt ]] && pip install -qr requirements.txt', user=project_folder)
 
             with virtualenv(venv):
                 with shell_env(DJANGO_SETTINGS_MODULE="{}.settings.{}".format(
                     project_folder,
                     remote['server'].get('settings_file', 'production')
                 )):
-
+                    sudo('pip install -q gunicorn', user=project_folder)
+                    
```

Comment:
I think you've got a load of trailing whitespace here.  Will need to remove that before merging.